File: Homework/Objective-C/HW7/HW7/Kangaroo.m
//
//  Kangaroo.m
//  HW7
//
//  Created by Руслан on 18.09.16.
//  Copyright © 2016 Ruslan Timchenko. All rights reserved.
//

#import "Kangaroo.h"

@implementation Kangaroo

- (NSString*) run {
    return [NSString stringWithFormat:@"Kangaroo %@ is running", self.name];
}

- (NSString*) jump {
    return [NSString stringWithFormat:@"Kangaroo %@ is jumping", self.name];

}

@end

File: Homework/Swift/HW7/HW7/Kangaroo.swift
//
//  Kangaroo.swift
//  HW7
//
//  Created by Руслан on 19.09.16.
//  Copyright © 2016 Ruslan Timchenko. All rights reserved.
//

import Foundation

class Kangaroo: Runners, Jumpers {
    
    let name: String
    
    init(name: String) {
        self.name = name
    }
    
    func run() -> String {
        return "Kangaroo \(self.name) is running"
    }
    
    func jump() -> String {
        return "Kangaroo \(self.name) is jumping"
    }
    
    func laugh() -> String {
        return "Kangaroo \(self.name) doesn't laugh"
    }


}
